app/app/components/OurTeam.jsx

```diff
@@ -0,0 +1,102 @@
+"use client"; // Mark this component as a client component
+
+import React, { useEffect, useRef, useState } from "react";
+import Image from "next/image"; // Import Image from Next.js
+import styles from "./OurTeam.module.css"; // Import the CSS module
+
+const OurTeam = () => {
+    const [members, setMembers] = useState([]); // State to hold the member data
+    const [visibleMembers, setVisibleMembers] = useState([]); // State for visibility
+    const membersRef = useRef([]);
+
+    useEffect(() => {
+        // Fetch members from JSON file
```

Comment:
You can remove these comments

Reply:
When should we leave a comment?

Whenever we are doing something more complex or strange or out of the normal.
It is super difficult to master when you are starting, and it will get better over time.
But as a rule of thumb: if you are using just simple built in React/JS functions in a line, you don't need to add in a comment.